dbbackup/management/commands/mediabackup.py

```diff
@@ -95,6 +90,11 @@ def get_source_dir(self):
 
     def create_backup_file(self, source_dir, backup_basename, **kwargs):
         temp_dir = tempfile.mkdtemp(dir=dbbackup_settings.TMP_DIR)
+        # Amazon S3 data model is a flat structure. However, Amazon Console
```

Comment:
Could you give me the trace back of this error ?

> Le 29 août 2015 à 21:56, Sergey Zherevchuk <notifications@github.com> a écrit :
> 
> In dbbackup/management/commands/mediabackup.py:
> 
> > @@ -95,6 +90,11 @@ def get_source_dir(self):
> >  
> >      def create_backup_file(self, source_dir, backup_basename, **kwargs):
> >          temp_dir = tempfile.mkdtemp(dir=dbbackup_settings.TMP_DIR)
> > +        # Amazon S3 data model is a flat structure. However, Amazon Console
> Sorry, if I don't understand what you want to say. 
> / is usable (mb because I use it). When I try to add backup filename my_backup_folder/media.tar.gz I get an error due to have no dir my_backup_folder in tmp folder (on unix system, mb it's not actual for win platform). The simplest way to avoid it - add this trick. Mb os.makedirs will be better approach?
> 
> —
> Reply to this email directly or view it on GitHub.
>